BlueprintUI/Sources/Layout/Builder/StackChild.swift

```diff
@@ -0,0 +1,91 @@
+import UIKit
+
+/// `StackChild` is a wrapper which allows an element to define its StackLayout traits and Keys.
+/// This struct is particularly useful when working with `@ElementBuilder`.
+/// By default, elements will default to a nil key and the default `StackLayout.Traits` initializer.
+/// `@StackElementBuilder` will check every child to see if it can be type cast to a `StackChild`
+/// and then pull of the given traits and key and then apply those to the stack
+public struct StackChild {
+    public let element: Element
+    public var traits: StackLayout.Traits
+    public var key: AnyHashable?
+    
+    public struct Sizing {
+        public static let fixed: Self = .init(growPriority: 0, shrinkPriority: 0)
```

Comment:
Two others we could add here would be `grows` and `shrinks` with 0 and 1 as their values (and vice-versa).

Reply:
Does this look good to you? 
![image](https://user-images.githubusercontent.com/7925830/132033741-23bf33ca-2d50-4c0c-b395-cc318b65cd83.png)

Reply:
(Yes! Sorry for the slow reply..)